server/controllers/profile.js

```diff
@@ -131,7 +129,7 @@ exports.getProfileByUser = asyncHandler(async (req, res, next) => {
 //@desc find one profile with a particular ID and update the info within
 //access private
 exports.updateProfile = asyncHandler(async (req, res, next) => {
-  const { id } = req.params;
+  const { userId } = req.user;
```

Comment:
```suggestion
  const { id } = req.user;
```

I don't think the `req.user` object contains a `userId` prop :)